packages/neuron-wallet/src/services/wallets.ts

```diff
@@ -280,9 +282,11 @@ export default class WalletService {
   }
 
   // TODO: move this method and generateTx/sendTx out of this file
-  public static async updateUsedAddresses(addresses: string[]) {
+  public static async updateUsedAddresses(addresses: string[], anyoneCanPayBlake160s: string[]) {
     const addrs = await AddressService.updateTxCountAndBalances(addresses)
-    const walletIds: Set<string> = new Set(addrs.map(addr => addr.walletId))
+    const addressVersion = NetworksService.getInstance().isMainnet() ? AddressVersion.Mainnet : AddressVersion.Testnet
+    const anyoneCanPayAddrs = await AddressService.updateUsedByAnyoneCanPayByBlake160s(anyoneCanPayBlake160s, addressVersion)
```

Comment:
The name `updateUsedByAnyoneCanPayByBlake160s` is weird because the param is `anyoneCanPayBlake160s`

Reply:
`blake160s` used by anyone-can-pay

Reply:
> `blake160s` used by anyone-can-pay

I see, the name `updateUsedByAnyoneCanPayByBlake160s` is actually `update usedByAnyoneCanPay by blake160s` instead of `update usedBy(AnyoneCanPayByBlake160s)`